### ISSUE TITLE
table_name model setting is ignored

### ISSUE THREAD

**Issue by tamersalama** [lowjoel/activerecord-userstamp#6]
with a model set to map to a different table compared to the expected default (using `self.table_name = 'new_name'`) - the gem still attempt to load column information for original name.

```
    # stampable.rb
     ...

    def inherited(klass)
      super

      klass.class_eval do
        add_userstamp_associations({}) #this appears to ignore `table_name = <new_name>`
      end
    end
```

**Comment by lowjoel** [comment]
Yup, this is nasty. In the inherited callback, the class hasn't been defined yet. I'm still looking for ways to balance magic-ness and configurability, this is where one of those places where the magic of automatically finding the fields to create associations for falls flat.

**Comment by tamersalama** [comment]
I've noticed the `add_userstamp_association` call in another class method (`stampable`). Removing this `class_eval` block within `inherited` appears to correct the issue, at least in my case. I'm shooting from the hip as I haven't fully tested it or traced the calls.

**Comment by lowjoel** [comment]
Yes, that disables the magic creator/updater associations from being created, it isn't exactly a solution I'm aiming for...

**Comment by lowjoel** [comment]
@tamersalama Can you have a go at the PR in #12 and see if that helps? Thanks.

**Comment by lowjoel** [comment]
Should be fixed by #12.
